.azure-pipelines/client.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         Windows:
           OSVmImage: 'vs2017-win2016'
         MacOs:
-          OSVmImage : 'macOS-10.13'
```

Comment:
Is this space what broke the mac build?

Reply:
We thought it might be, but it doesn't appear so.  We now suspect DevOps is currently having issues running jobs on Mac agents.